### ISSUE TITLE
Multi-CPU kernel for sparse_tensor_dense_matmul

### ISSUE THREAD

**Issue by rueberger** [tensorflow/tensorflow#4337]
Are there plans to implement a multi-cpu kernel for tf.sparse_tensor_dense_matmul? The current version seems to be single core only and is not performing well. 

I am working with a very large sparse tensor - about 8GB sparse and 200GB dense that I cannot work with as a dense tensor for memory reasons. Any suggestions for achieving fast matrix multiplication?

**Comment by jmchen-g** [comment]
@concretevitamin Could you take a look at this? Thanks.

**Comment by concretevitamin** [comment]
@ebrevdo is a more suitable person to look at this, I believe.

**Comment by drpngx** [comment]
Is this still current?

**Comment by gunan** [comment]
Ping!
Is this still an issue?
@concretevitamin @ebrevdo could you comment and/or close this issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assigneee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yifeif** [comment]
@rueberger is this still an issue for you?

**Comment by rueberger** [comment]
Whoops sorry!! bouncehouse-bot is me...  forgot to log out of it...

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ebrevdo** [comment]
Agreed it would be nice to optimize these methods.  I'm not sure the COO representation is great for this, but we would review performance improvements to this op.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by mohantym** [comment]
Hi @rueberger! we are checking to see if you still need help in this issue , Have you tried latest stable version TF 2.6  yet? Please create a new issue if the issue is replicating in newer versions.   Feel free to  look up on [this thread ](https://stackoverflow.com/questions/38836269/does-tensorflow-view-all-cpus-of-one-machine-as-one-device)for  an answer.Thanks!

**Comment by rueberger** [comment]
Same answer as last time. No longer working on this and have not tried latest

**Comment by mohantym** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
